### PR TITLE
Add requirment

### PR DESCRIPTION
**Pull request by eng-mohamed-eldeeb**
Create book's actions and reducer and categories actions and reducer using the ducks pattern. Configure the Redux Store. Add an action for adding and removing the books, and one for checking the status.

### CODE REVIEW

File: src/redux/books/books.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
+export const ADD_BOOK = 'addBook';
+export const REMOVE_BOOK = 'removeBook';
```
Comment (er-contreras) [inline src/redux/books/books.js]: - [x] The value of each action type needs to follow the [ducks pattern](https://github.com/erikras/ducks-modular-redux) 😉

File: src/redux/categories/categories.js [hunk 2]
```diff
@@ -0,0 +1,8 @@
+const CHECK_STATUS = 'CHECK_STATUS';
```
Comment (er-contreras) [inline src/redux/categories/categories.js]: - [x] The value of action type needs to follow the [ducks pattern](https://github.com/erikras/ducks-modular-redux)

**Review by er-contreras — CHANGES_REQUESTED** [review]
Hi @eng-mohamed-eldeeb, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

almost there

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/redux/books/books.js [hunk 3]
```diff
@@ -0,0 +1,37 @@
+export const ADD_BOOK = 'bookStore/src/redux/book/ADD_BOOL';
+export const REMOVE_BOOK = 'bookStore/src/redux/book/REMOVE_BOOK';
```
Comment (mikethreels) [inline src/redux/books/books.js]: - [x] in line 1 there is a small typo, it should be `ADD_BOOK` instead of `ADD_BOOL`

**Review by mikethreels — CHANGES_REQUESTED** [review]
Hi El-deeb, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
### Because it wasn't mentioned in the previous review the following is OPTIONAL:
- [ ] please make sure to use[ the template](https://github.com/microverseinc/readme-template) for the readme file provided by Microverse, including the links (for example to the MIT license). keep in mind that all sections of the template are mandatory except for the "Live Demo", "Run Tests" and "Deployment" sections, these are only mandatory if the specific project you're working on requires it.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Amrhub — APPROVED** [review]
# Approved 💯 ✅ 🥳 🎈 🎉 

Hi @eng-mohamed-eldeeb, 👋🏻 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights ✅ 
Highly organized and clean code ✔️ 👨🏻‍💻 
Professional documentation ✔️ 
Correct workflow ✔️ 
Meaningful commit messages ✔️ 


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**